### ISSUE TITLE
Add trailing comment in CHECK and TAKETHIS responses

### ISSUE THREAD

**Issue by rra** [InterNetNews/inn#161]
**Reported by iulius on 10 Apr 2018 20:21 UTC**
Responses to CHECK and TAKETHIS commands currently do not mention the reason of a reject (notably from Python or Perl filter hook, but also directly from innd main checks).

On the contrary, responses to IHAVE mention the reason, but wrongly compute the size of the buffer response.  Computation of cp->Sendid.size should then be fixed.

**Comment by rra** [comment]
**Modified by iulius on 16 May 2018 19:57 UTC**

**Comment by rra** [comment]
**Comment by iulius on 21 May 2018 12:41 UTC**
In [10286](changeset:"10286"):
```CommitTicketReference repository="" revision="10286"
Add trailing comment in CHECK and TAKETHIS responses

Responses to CHECK and TAKETHIS commands did not mention the reason of
a reject (notably from Python or Perl filter hook, but also directly
from innd main checks).  This is now fixed.

closes #148
```
